pandas/core/arrays/arrow/array.py

```diff
@@ -569,11 +569,8 @@ def argsort(
     ) -> np.ndarray:
         order = "ascending" if ascending else "descending"
         null_placement = {"last": "at_end", "first": "at_start"}.get(na_position, None)
-        if null_placement is None or pa_version_under7p0:
-            # Although pc.array_sort_indices exists in version 6
-            # there's a bug that affects the pa.ChunkedArray backing
-            # https://issues.apache.org/jira/browse/ARROW-12042
-            fallback_performancewarning("7")
+        if null_placement is None:
+            fallback_performancewarning()
```

Comment:
is the warning still necessary?

Reply:
Good catch. I think now this can raise an error directly